### ISSUE TITLE
Multiple instances on page causes error

### ISSUE THREAD

**Issue by AshCoolman** [mishguruorg/angular-timezone-selector#9]
Hi, cool component. Just a heads - I get an error in the browser console when I have two selectors instanced.
- ng. v1.3.15
- angular-timezone-selector v1.2.1

``` jade
  .row
    .col-lg-12
      timezone-selector(ng-model="timezoneOne"
        display-utc="true"
        sort-by="offset"
        show-local="true"
        primary-choices="UTC GB WET GMT Asia/Macau")
    .col-lg-12
      timezone-selector(ng-model="timezoneTwo"
        display-utc="true"
        sort-by="offset"
        show-local="true"
        primary-choices="UTC GB WET GMT Asia/Macau")
```

Error below:

```
TypeError: Cannot read property 'nOffset' of undefined
    at link (http://localhost:8081/.build/js/vendor.min.js:88100:40)
    at http://localhost:8081/.build/js/vendor.min.js:30180:44
    at invokeLinkFn (http://localhost:8081/.build/js/vendor.min.js:30186:9)
    at nodeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29696:11)
    at compositeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29045:13)
    at compositeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29048:13)
    at compositeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29048:13)
    at nodeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29691:24)
    at compositeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29045:13)
    at nodeLinkFn (http://localhost:8081/.build/js/vendor.min.js:29691:24) <select style="min-width:300px;;min-width:300px;" ng-model="timezone" display-utc="true" sort-by="offset" show-local="true" primary-choices="UTC GB WET GMT Asia/Macau" class="ng-isolate-scope">(anonymous function) @ angular.js:11655(anonymous function) @ angular.js:8596invokeLinkFn @ angular.js:8260nodeLinkFn @ angular.js:7768compositeLinkFn @ angular.js:7117compositeLinkFn @ angular.js:7120compositeLinkFn @ angular.js:7120nodeLinkFn @ angular.js:7763compositeLinkFn @ angular.js:7117nodeLinkFn @ angular.js:7763(anonymous function) @ angular.js:7998processQueue @ angular.js:13248(anonymous function) @ angular.js:13264Scope.$eval @ angular.js:14466Scope.$digest @ angular.js:14282Scope.$apply @ angular.js:14571bootstrapApply @ angular.js:1455invoke @ angular.js:4203doBootstrap @ angular.js:1453bootstrap @ angular.js:1473angularInit @ angular.js:1367(anonymous function) @ angular.js:26304fire @ jquery.js:3099self.fireWith @ jquery.js:3211jQuery.extend.ready @ jquery.js:3417completed @ jquery.js:3433
```

Thanks

**Comment by karlhaas** [comment]
Had the same issue and provided pull request #12 which should fix it.

**Comment by ashokfernandez** [comment]
Fixed in https://github.com/mishguruorg/angular-timezone-selector/pull/12 thanks @karlhaas! I've added your name to the contributors in the readme

**Comment by ashokfernandez** [comment]
Released as v1.2.4
